go.mod

```diff
@@ -70,18 +54,22 @@ require (
 	github.com/ltcsuite/ltcwallet/wallet/txsizes v1.1.0
 	github.com/ltcsuite/ltcwallet/walletdb v1.3.5
 	github.com/ltcsuite/ltcwallet/wtxmgr v1.5.0
+	github.com/nxadm/tail v1.4.8
+	github.com/onsi/ginkgo v1.15.0
+	github.com/onsi/gomega v1.10.5
+	github.com/yeqown/go-qrcode v1.5.1
+	go.etcd.io/bbolt v1.3.7
+	golang.org/x/crypto v0.7.0
 	golang.org/x/exp/shiny v0.0.0-20220827204233-334a2380cb91
+	golang.org/x/image v0.0.0-20220722155232-062f8c9fd539
+	golang.org/x/sync v0.1.0
+	golang.org/x/text v0.8.0
 )
 
 require (
-	decred.org/cspp/v2 v2.0.0 // indirect
-	decred.org/dcrwallet v1.7.1
-	github.com/companyzero/sntrup4591761 v0.0.0-20200131011700-2b0d299dbd22 // indirect
-)
-
-require github.com/crypto-power/instantswap v0.0.0-20230619161136-95ecf47d5ebf
-
-require (
+	decred.org/cspp/v2 v2.1.0 // indirect
+	decred.org/dcrwallet v1.7.0 // indirect
```

Comment:
@dmigwi why is this on 1.7.0

Reply:
That is automatically imported by the dependencies. Let me look at which dependency is causing this indirect use.

Reply:
It turns out that dcrdata hasn't been updated we use it a dependency. https://github.com/decred/dcrdata/tags

 I will try and phase it out.